pkgs/development/tools/build-managers/bob/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "bob";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "benchkram";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-JG1fysCqqd/MwpNhKJwLr4cTGq4/88f9OMMapb+r3bc=";
+  };
+
+  CGO_ENABLED = 0;
+
```

Comment:
Is there a reason to build this statically?
```suggestion
```

Reply:
I mean, apart from following what upstream [does](https://github.com/benchkram/bob/blob/main/.goreleaser.yml#L6). Upstream is interested in being able to distribute the pre-compiled binary to different Linux distros, but we are not necessarily into that, so I think it would be better to go with the default and remove this line.